index.js

```diff
@@ -191,3 +178,5 @@ function copy (a, b, cb) {
     })
   })
 }
+
+module.exports = prebuildify
```

Comment:
could you move this up before the `prebuildify` function but after the requires?